servers/src/common/types.rs

```diff
@@ -248,6 +248,20 @@ pub struct WebHooksConfig {
 	pub block_received_url: Option<String>,
 	/// url to POST block data when a new block is accepted by our node (might be a reorg or a fork)
 	pub block_accepted_url: Option<String>,
+	/// number of worker threads in the tokio runtime
+	#[serde(default = "default_nthreads")]
+	pub nthreads: u16,
+	/// timeout in seconds for the http request
+	#[serde(default = "default_timeout")]
+	pub timeout: u16,
+}
+
+fn default_timeout() -> u16 {
```

Comment:
why do we need it as a function?

Reply:
That's how serde works, you can't write `#[serde(default = "10")]` it needs a function name. https://serde.rs/attr-default.html